.travis.yml

```diff
@@ -0,0 +1,11 @@
+dist: trusty
+language: java
+jdk:
+- oraclejdk8
+script:
+- if [ "$TRAVIS_BRANCH" = "master" ] && [ "$TRAVIS_PULL_REQUEST" = "false" ]; then
+  mvn deploy --settings maven-settings.xml; else mvn install; fi
+env:
+  global:
```

Comment:
@solemabrothers delete these environment variables and create new ones using the steps https://wiki.openmrs.org/display/docs/Setup+Travis+CI+to+Deploy+Snapshots+to+Nexus